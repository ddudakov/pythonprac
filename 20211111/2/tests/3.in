1,2,3,4,5,6
(1,1),(2,2),(11,11)
(0,0.0),(3.0,0),(0,4)
