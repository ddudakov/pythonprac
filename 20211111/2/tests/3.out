Invalid input
Not a triangle
6.00
